oauth2client/file.py

```diff
@@ -20,12 +20,17 @@
 
 import os
 import threading
+import warnings
 
 from oauth2client import client
 
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
+_SYM_LINK_MESSAGE = 'File: {0}: Is a symbolic link.'
+_IS_DIR_MESSAGE = '{0}: Is a directory'
+_MISSING_FILE_MESSAGE = 'Cannot access {0}: No such file or directory'
+
 
 class CredentialsFileSymbolicLinkError(Exception):
```

Comment:
I went ahead and removed it, in favor of `IOError`.  When grepping for that exception, I found a duplicated `validate_file` function, so I moved `validate_file` to `oauth2client.util` and used that instead.